txircd/modules/rfc/mech_oper.py

```diff
@@ -24,6 +25,32 @@ def actions(self):
 	
 	def userModes(self):
 		return [ ("o", ModeType.NoParam, self) ]
+
+	def verifyConfig(self, config):
+		if "opers" in config:
+			if not isinstance(config["opers"], dict):
+				raise ConfigValidationError("opers", "value must be a dictionary")
+			for operNick, values in config["opers"].iteritems():
+				if "password" not in values:
+					raise ConfigValidationError("opers", "no password defined for oper \"{}\"".format(operNick))
+				if not isinstance(values["password"], basestring):
+					raise ConfigValidationError("opers", "value \"password\" for oper \"{}\" must be string".format(operNick))
+				if "hash" in values and values["hash"] != "pbkdf2": # Change later if we add more hashing modules
```

Comment:
Can we not make assumptions about other modules here?'
We should not make assumptions about other modules (especially non-core modules like those in stringhash).